scripts/run_lighthouse_tests.py

```diff
@@ -31,8 +31,20 @@
 
 WEBPACK_BIN_PATH = os.path.join(
     common.CURR_DIR, 'node_modules', 'webpack', 'bin', 'webpack.js')
+LIGHTHOUSE_MODE_PERFORMANCE = 0
+LIGHTHOUSE_MODE_ACCESSIBILITY = 1
+SERVER_MODE_PROD = 2
+SERVER_MODE_DEV = 3
 GOOGLE_APP_ENGINE_PORT = 8181
 SUBPROCESSES = []
+LIGHTHOUSE_CONFIG_PATHS = {
+    LIGHTHOUSE_MODE_PERFORMANCE: '--config=.lighthouserc.js',
```

Comment:
Only the filename (.lighthouserc.js) should be here.

(Maybe rename to LIGHTHOUSE_CONFIG_FILENAMES instead)